core/server/src/main/java/alluxio/worker/WorkerContext.java

```diff
@@ -65,29 +45,16 @@ public static WorkerNetAddress getNetAddress() {
   }
 
   /**
-   * Sets {@link WorkerNetAddress} object of this worker.
-   *
    * @param netAddress {@link WorkerNetAddress} object of this worker
    */
   public static void setWorkerNetAddress(WorkerNetAddress netAddress) {
     sNetAddress = netAddress;
   }
 
   /**
-   * Resets the worker context, for test only.
-   * TODO(binfan): consider a better way to mock test configuration
+   * Resets the worker context.
    */
   public static void reset() {
```

Comment:
I think we can drop this reset method now, or at least use it very rarely (I don't think tests care about the worker source).